phpunit/functional/Group_UserTest.php

```diff
@@ -213,4 +213,48 @@ public function testIsUserInGroup()
         $this->assertTrue(\Group_User::isUserInGroup(getItemByTypeName('User', 'tech', true), $groups_id));
         $this->assertFalse(\Group_User::isUserInGroup(getItemByTypeName('User', 'glpi', true), $groups_id));
     }
+
+    public function testDeleteUserDefaultGroup()
+    {
+        $group = new \Group();
+        $gid = (int)$group->add([
+            'name' => 'Test group'
+        ]);
+        $this->assertGreaterThan(0, $gid);
+
+        $user = getItemByTypeName('User', 'tech');
+
+        $group_user = new \Group_User();
+        $this->assertGreaterThan(
+            0,
+            (int)$group_user->add(
+                [
+                    'groups_id' => $gid,
+                    'users_id'  => $user->getID()
+                ]
+            )
+        );
+
+        $this->assertTrue(
+            $user->update(
+                [
+                    'id' => $user->getID(),
+                    'groups_id' => $gid
+                ]
+            )
+        );
+
```

Comment:
Is that update really needed?
```suggestion
        $this->assertTrue(
            $user->update(
                [
                    'id' => $user->getID(),
                    'groups_id' => $gid
                ]
            )
        );

```

Reply:
yes, to set user default group =)

This is the link between the user and the group that was added previously, and this action does not update the default group